### PR TITLE
update documentation

### LINKED ISSUE #30: репозиторий с deb пакетами
Не баг, 
просто хочу сообщить что сделал репозиторий с пакетами для Ubuntu но должно подойти и для Debian/

https://launchpad.net/~linvinus/+archive/ubuntu/rhvoice/

### PR DESCRIPTION
**Pull request by alex19EP**
update documentation.

* migrate from wiki
  * Privacy Policy
  * Linux compilation documentation
  * configuration documentation
  * links to prebuilt binaries
* update readme
  * make readme a markdown doc
  * add a proper license file
* replace installation document
* add doc about packaging status closes #30

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# RHVoice
+
+RHVoice is a free and open source speech synthesizer.
+
+## Features
+
+### Speech synthesis method
+
+RHVoice uses statistical parametric synthesis. It relies on existing
```
Comment (techtonik) [inline README.md:9]:
```suggestion
RHVoice uses [statistical parametric synthesis](https://en.wikipedia.org/wiki/Speech_synthesis#HMM-based_synthesis).
It relies on existing
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+* Brazilian Portuguese
+* Esperanto
+* Georgian
+* Ukrainian
+* Kyrgyz
+* Tatar
+
+In theory, it is possible to implement support for
+other languages, if all the necessary resources can be found or
+created.
```
Comment (techtonik) [inline README.md:34]: Quote obvious without a link to the necessary resources.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+* Windows
+* GNU/Linux
+* Android
+
+It is compatible with
+standard text-to-speech interfaces on these platforms: SAPI5 on
+Windows, [Speech Dispatcher](http://devel.freebsoft.org/speechd) on
+GNU/Linux and Android's text-to-speech APIs. It can also be used by
+the [NVDA screen reader](http://www.nvaccess.org) directly (the driver
+is provided by RHVoice itself).
```
Comment (techtonik) [inline README.md:49]: I would like to read more about Linux text-to-speech API from here, because I don't know.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+## license
```
Comment (techtonik) [inline README.md:51]:
```suggestion
## License
```

File: LICENSE.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
+# licenses
+
+The main library is distributed under [LGPL v2.1](https://www.gnu.org/licenses/lgpl-2.1.html) or later. But it
+relies on [MAGE](https://github.com/numediart/mage) for better responsiveness. MAGE is distributed under
+[GPL v3](http://www.gnu.org/licenses/gpl-3.0.html) or later, so the combination is under GPL v3 or later. If you
+want to use the library in your program under [GPL v2](http://www.gnu.org/licenses/gpl-2.0.html) or LGPL, compile the
+library without MAGE.
```
Comment (techtonik) [inline LICENSE.md:7]: Need an instruction how to do this.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+The main library is distributed under [LGPL v2.1](https://www.gnu.org/licenses/lgpl-2.1.html) or later.
+some components and voices use other licenses.for more information please read [license.md](license.md) file.
```
Comment (techtonik) [inline README.md:54]: The library is distributed under GPL, because it ships binaries compiled with MAGE.

Comment (techtonik) [reply README.md:54]: However, the sources of the library are using LGPL license.

File: README.md (hunk 2 above)
Comment (alex19EP) [reply README.md:34]: @Olga-Yakovleva currently working on documentation and voice creation. so link will be added later.

File: README.md (hunk 3 above)
Comment (alex19EP) [reply README.md:49]: what about

> [Speech Dispatcher](http://devel.freebsoft.org/speechd)

line 46

File: LICENSE.md (hunk 5 above)
Comment (alex19EP) [reply LICENSE.md:7]: not here.

**Comment by alex19EP** [comment]
> 
> 
> Awesome. I would probably avoid extra dirs in `docs/`, but it doesn't matter.

I think it looks more organized. I just don’t like it when everything is on the heap.

File: README.md (hunk 3 above)
Comment (techtonik) [reply README.md:50]: I misread "on GNU/Linux" as "or GNU/Linux". Need to be more attentive with fast reviews. =)

**Pull request merged**